docs/user-guide/README.md

```diff
@@ -18,13 +18,18 @@ npm install --save-dev eslint eslint-plugin-regexp
 <!--USAGE_SECTION_START-->
 
 Add `regexp` to the plugins section of your `.eslintrc` configuration file (you can omit the `eslint-plugin-` prefix)  
-and either use the recommended configuration or configure the rules you want:
+and either use one of the two configurations available (`recommended` and `all` ) or configure the rules you want:
 
 ### The recommended configuration
 
 The `plugin:regexp/recommended` config enables a subset of [the rules](../rules/README.md) that should be most useful to most users.
 *See [https://github.com/ota-meshi/eslint-plugin-regexp/blob/master/lib/configs/recommended.ts](https://github.com/ota-meshi/eslint-plugin-regexp/blob/master/lib/configs/recommended.ts) for more details.*
 
+### The all configuration
+
+The `plugin:regexp/all` config enables all rules. It's meant for testing, not for production use because it changes with every minor and major version of the plugin. Use it at your own risk.
+  See [https://github.com/ota-meshi/eslint-plugin-regexp/blob/master/lib/configs/all.ts](https://github.com/ota-meshi/eslint-plugin-regexp/blob/master/lib/configs/all.ts) for more details.*
+
 ```js
```

Comment:
This code block should be in the section of the recommended configuration.

I don't think we need a code example for this config. We already have this example for the recommended config and people should be able to figure out which word to replace with `all`.